Gemfile.lock

```diff
@@ -1,38 +1,38 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    actioncable (6.0.2.2)
-      actionpack (= 6.0.2.2)
+    actioncable (6.0.3.1)
+      actionpack (= 6.0.3.1)
       nio4r (~> 2.0)
       websocket-driver (>= 0.6.1)
-    actionmailbox (6.0.2.2)
-      actionpack (= 6.0.2.2)
-      activejob (= 6.0.2.2)
-      activerecord (= 6.0.2.2)
-      activestorage (= 6.0.2.2)
-      activesupport (= 6.0.2.2)
+    actionmailbox (6.0.3.1)
+      actionpack (= 6.0.3.1)
+      activejob (= 6.0.3.1)
+      activerecord (= 6.0.3.1)
+      activestorage (= 6.0.3.1)
+      activesupport (= 6.0.3.1)
       mail (>= 2.7.1)
-    actionmailer (6.0.2.2)
-      actionpack (= 6.0.2.2)
-      actionview (= 6.0.2.2)
-      activejob (= 6.0.2.2)
+    actionmailer (6.0.3.1)
+      actionpack (= 6.0.3.1)
+      actionview (= 6.0.3.1)
+      activejob (= 6.0.3.1)
       mail (~> 2.5, >= 2.5.4)
       rails-dom-testing (~> 2.0)
-    actionpack (6.0.2.2)
-      actionview (= 6.0.2.2)
-      activesupport (= 6.0.2.2)
+    actionpack (6.0.3.1)
```

Comment:
### [actionpack](https://rubyonrails.org)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 6.0.2.2 → 6.0.3.1

[_[change-log](https://github.com/rails/rails/blob/v6.0.3.1/actionpack/CHANGELOG.md), [source-code](https://github.com/rails/rails/tree/v6.0.3.1/actionpack)_]

:tada: Patched vulnerabilities:

- [CVE-2020-8166](https://nvd.nist.gov/vuln/detail/CVE-2020-8166)
  **Ability to forge per-form CSRF tokens given a global CSRF token**

  
  URL: https://groups.google.com/forum/#!topic/rubyonrails-security/NOjKiGeXUgw

  It is possible to possible to, given a global CSRF token such as the one present in the authenticity_token meta tag, forge a per-form CSRF token for any action for that session.  Versions Affected:  rails < 5.2.5, rails < 6.0.4 Not affected:       Applications without existing HTML injection vulnerabilities. Fixed Versions:     rails >= 5.2.4.3, rails >= 6.0.3.1  Impact ------  Given the ability to extract the global CSRF token, an attacker would be able to construct a per-form CSRF token for that session.  Workarounds -----------  This is a low-severity security issue. As such, no workaround is necessarily until such time as the application can be upgraded.


- [CVE-2020-8164](https://nvd.nist.gov/vuln/detail/CVE-2020-8164)
  **Possible Strong Parameters Bypass in ActionPack**

  
  URL: https://groups.google.com/forum/#!topic/rubyonrails-security/f6ioe4sdpbY

  There is a strong parameters bypass vector in ActionPack.  Versions Affected:  rails <= 6.0.3 Not affected:       rails < 4.0.0 Fixed Versions:     rails >= 5.2.4.3, rails >= 6.0.3.1  Impact ------ In some cases user supplied information can be inadvertently leaked from Strong Parameters.  Specifically the return value of `each`, or `each_value`, or `each_pair` will return the underlying "untrusted" hash of data that was read from the parameters.  Applications that use this return value may be inadvertently use untrusted user input.  Impacted code will look something like this:  ``` def update   # Attacker has included the parameter: `{ is_admin: true }`   User.update(clean_up_params) end  def clean_up_params    params.each { |k, v|  SomeModel.check(v) if k == :name } end ```  Note the mistaken use of `each` in the `clean_up_params` method in the above example.  Workarounds ----------- Do not use the return values of `each`, `each_value`, or `each_pair` in your application.






<details>
<summary>Commits</summary>

A change of **257** commits. See the full changes on [the compare page](https://github.com/rails/rails/compare/v6.0.2.2...v6.0.3.1).

These are the first 10 commits:
- (e5310b5) [Merge pull request #37849 from kamipo/fix_since_and_ago](https://github.com/rails/rails/commit/e5310b5d745b63ca6a344508390d9d3920dff379)
- (21a1b18) [Revert "registers previous commit in the CHANGELOG"](https://github.com/rails/rails/commit/21a1b187df474d8aa8f75d776bdeefd6c91a58f1)
- (67b7923) [Revert "let environments configure load-related paths"](https://github.com/rails/rails/commit/67b79238c4ab320427f03486c0b9b525f7b53e44)
- (dc84717) [Add file analysis to the Active Storage guide [ci skip]](https://github.com/rails/rails/commit/dc847176647a3f377ef4d3065921b82ca8019eea)
- (675f967) [Controller can be symbols as well](https://github.com/rails/rails/commit/675f967a314c3029137d1ed6a306c85d5a4a958f)
- (480745c) [Merge pull request #37862 from giraffate/fix_random_ci_failu…](https://github.com/rails/rails/commit/480745c56752fb381ba0b84b59eb58cec62eb72e)
- (596e125) [Merge pull request #37866 from giraffate/correct_default_val…](https://github.com/rails/rails/commit/596e1250cfde2768232432071365a10554517c9c)
- (24bf3f0) [Merge pull request #37882 from kamipo/should_keep_zero_durat…](https://github.com/rails/rails/commit/24bf3f08c096d1cbb934a0265cb707a8d3f9bfc2)
- (d2f53ac) [Switch to the writer DB to generate ASt previews](https://github.com/rails/rails/commit/d2f53ac5c306557c3d3204399b34fccfbd4d6baa)
- (e193acb) [Merge pull request #37666](https://github.com/rails/rails/commit/e193acb118b2bbcb4fa06d727f63a26e62405d9f)


</details>